api/server/routes/reviews.js

```diff
@@ -80,20 +65,18 @@ module.exports = (fastify, opts, next) => {
   })
 
   // Create new review
-  fastify.post('/:videoUrl', { schema: opts.schemas.add }, async (req, reply) => {
+  fastify.post('/', { schema: opts.schemas.add }, async (req, reply) => {
     let review = new Review(req.body.review)
 	const err = await review.save((err) => { return !!err })
-	
 	reply.send({ err })
   })
 
-  // // Increment vote of a review
-  // fastify.post('/:reviewId', { schema: opts.schemas.add }, async (req, reply) => {
-  //   let review = new Review(req.body.review)
-  //   review.save((err) => {
-  //     reply.send({ err: !!err })
-  //   })
-  // })
+  // Increment vote of a review
+  fastify.post('/:reviewId', async (req, reply) => {
+      const data = await Review.findByIdAndUpdate(req.params.reviewId , {$inc: {vote: req.body.review.vote}})
+    req.log.info('   ====== data.vote: ' + data.vote+ " ========")
```

Comment:
You need to use single-quotes only to comply with standard JS

Reply:
ok Thanks ;)